packages/functional-tests/tests/misc/keyStretchingV2.spec.ts

```diff
@@ -2,11 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { test, expect } from '../../lib/fixtures/standard';
-import AuthClient from 'fxa-auth-client/browser';
+import { test, expect, PASSWORD } from '../../lib/fixtures/standard';
```

Comment:
I've never liked the way we have the constant passsword... I'd much rather it be generated like email, and/or have some variation.